vmdb/app/controllers/application_controller.rb

```diff
@@ -2168,6 +2168,10 @@ def prov_redirect(typ=nil)
         render_flash_not_applicable_to_model(typ)
         return
       end
+      if typ == "migrate" && !VmOrTemplate.supports_migrate?(vm_ids)
```

Comment:
is_available? method does not take list of ids as parameter but in the case above supports_migrate? method is taking list of ids as params and then checking if any of the selected VMs do not support migrate task, not sure how would i do the same using is_available? method.